lib/ui/page/home/page/chat/widget/message_timestamp.dart

```diff
@@ -77,6 +77,30 @@ class MessageTimestamp extends StatelessWidget {
     final bool isError = status == SendingStatus.error;
     final bool isSending = status == SendingStatus.sending;
 
+    SvgData getIcon() {
```

Comment:
Shouldn't this method be a class level private method? And it might return `SvgData?` so that when it's null, then `if (isSent || isDelivered || isRead || isSending || isError) ...` might be replaced with `if (icon != null) ...`?